class ConflictError(Exception):
    pass


class ValidateError(Exception):
    pass


class NotFoundError(Exception):
    pass
